Core/NoOpRunner.Core/GameSettings.cs

```diff
@@ -7,6 +9,10 @@ public  static class GameSettings
 
         private const int CellsSizeMultiplier = 4;
 
+        public static readonly TimeSpan MoveAnimationDuration = TimeSpan.FromMilliseconds(1000 / 20);
+        
+        public static readonly TimeSpan Fps = TimeSpan.FromMilliseconds(70);
```

Comment:
**[nitpick]** misleading name, this is time between frames, not frames per second